### ISSUE TITLE
visualstudio2022buildtools has version 117.0.0.0

### ISSUE THREAD

**Issue by RexMorgan** [jberezanski/ChocolateyPackages#122]
Should this be version 17.0.0.0, instead?

**Comment by jberezanski** [comment]
This package version (117.0.0.0) is chosen deliberately. Because of how the VS installer works, the package always installs the latest released Visual Studio version (or upgrades an installed VS 2022 instance to the latest version).

If/when, however, I find time and motivation to create "fixed version" packages (which are able to install the specific VS version, using the [special installers published by Microsoft](https://docs.microsoft.com/en-us/visualstudio/releases/2022/release-history#fixed-version-bootstrappers)), those package versions will correspond to the actual VS versions installed by the packages.

Summing up, the idea for VS 2022 package version numbers is:
- major version greater than 100, e.g. 1xx.y.z.0: installs the latest VS version (using the install bootstrapper* released with VS xx.y.z); upgrades an existing VS installation to the latest version
- major version less than 100, e.g. xx.y.z.0: installs VS xx.y.z precisely; does not support updating an existing VS installation

(\* the bootstrapper is a small program which starts the process of installing Visual Studio - its main task is to download and install the proper Visual Studio Installer program and invoke it to perform the actual installation)

**Comment by RexMorgan** [comment]
Thanks for the clarification!

**Comment by jberezanski** [comment]
Related: #82

**Comment by jberezanski** [comment]
Can we close this issue or do you have any additional questions?
